code/modules/projectiles/guns/beam_rifle.dm

```diff
@@ -472,7 +472,7 @@
 		if(original && (original.layer>=2.75) || ismob(original))
```

Comment:
does anyone have any clue what this code is? it's in all the projectile code and calls bump during bump based on some layer checks of a a certain object.